src/apis/apis.type.ts

```diff
@@ -0,0 +1,28 @@
+export type Obj = Record<string, unknown>;
+
+export type StringObj = Record<string, string>;
+
+export type OrderBy = "recent" | "favorite";
+
+export interface GetProductsParams extends Obj {
```

Comment:
P1:
해당 코드의 의미는 `Obj`에서 확장한 타입을 정의하는 것으로 이렇게 작성하시면
아래에 적어주신 page, pageSize, orderBy, keyword 외에도 다른 값들도 받을 수 있게 열린 타입이 됩니다.
아래처럼 작성하시는 것을 추천드립니다.

```suggestion
export interface GetProductsParams {
```